### DISCUSSION TITLE
PI 12 Iteration 2

### DISCUSSION THREAD

**Discussion by mkanal** [eclipse-tractusx/item-relationship-service#417] [General]
GOAL: 

## Minister Security
@mkanal 
Timebox: 2h
- [ ] Check Veracode for findings
- [ ] Check Invict for findings
- [x] run and check Dashtool IP

## Release Champion: 
@dsmf 
Timebox: 2h

- [ ] Has the Release been conducted (Release is being skipped!)
- [ ] Check for Dependabots PR's on Tractus-X

## Docu-Guru:
@ds-jhartmann 
Timebox: 2h 
- [ ] Focused on Documentation of Tickets

## Testmanager
@ds-alexander-bulgakov 
- [ ] Update [Release 24.03 ---- Content Overview](https://confluence.catena-x.net/display/CXAR/Release+24.03+----+Content+Overview)
- [ ] Update Test Status - PI11 [Test Status - PI11](https://confluence.catena-x.net/display/PL/Test+Status+-+PI11)
- [ ] Update [Integration Test - PI12 - Release 2024.05](https://confluence.catena-x.net/display/PL/Integration+Test+-+PI12+-+Release+2024.05)

## Firefighter
@ds-ext-kmassalski  
- [ ] Maintenance of our community
- [ ] Response to Github issues https://github.com/eclipse-tractusx/item-relationship-service/issues
- [ ]  To be defined, eg: Onboarding of new colleague, quality-gates, escalation topics
- [ ] Communicating and discussing with external contributors in Eclipse Matrix Chat

**Comment by ds-alexander-bulgakov** [comment]
- [x] Update [Release 24.03 ---- Content Overview](https://confluence.catena-x.net/display/CXAR/Release+24.03+----+Content+Overview)
- [x]  Update [Test Status - PI11](https://confluence.catena-x.net/display/PL/Test+Status+-+PI11) Test Status - PI11
- [x]  Update [Integration Test - PI12 - Release 2024.05](https://confluence.catena-x.net/display/PL/Integration+Test+-+PI12+-+Release+2024.05)
